cmd/download_test.go

```diff
@@ -238,9 +235,9 @@ const payloadTemplate = `
 		},
 		"file_download_base_url": "%s",
 		"files": [
-		"%s",
-		"%s",
-		"%s"
+		"/file-1.txt",
```

Comment:
How do you feel about moving the constants, template, and fakeDownServer declarations to the top of the file? So reading the test file from top to bottom give a complete picture of what those variables are. 

Right now I have to scan towards the bottom of the file to see what requestorSelf is or even what we are doing with the template.

Reply:
I don't much like the idea, to be honest. I don't want to have to scan through a couple hundred lines of code before we get to the tests.

Reply:
Thanks! And thanks for taking the time to ask the question. I didn't know I didn't like it until you asked!